tests/std/tests/P0355R7_calendars_and_time_zones_dates/test.cpp

```diff
@@ -28,13 +28,30 @@ constexpr bool day_test() {
     assert(static_cast<unsigned>(d1) == 0);
 
     day d2{0u};
-    assert(d1 == d2);
+    assert(d1 == d2++);
+    assert(d1 < d2);
+    assert(d2 > d1);
+
+    day d3{0};
+    assert(!d3.ok());
+    ++d3;
+    for (int i = 1; i <= 31; ++i, ++d3) {
+        assert(d3.ok());
+    }
+    assert(!d3.ok());
+
+    const auto d4 = 0d;
+    assert(d1 == d4);
+    static_assert(is_same_v<day, decltype(0d)>, "0d is not chrono::day");
+    // static_assert(is_same_v<day, decltype(d4)>, "d4 is not chrono::day");

```

Comment:
Maybe I'm missing something obvious but why would this test fail? Even changing `d4` to explicitly be `chrono::day`.